Package.swift

```diff
@@ -21,12 +24,6 @@ let package = Package(
                 .product(name: "Fluent{{fluent.db.module}}Driver", package: "fluent-{{fluent.db.url}}-driver"),{{/fluent}}{{#leaf}}
                 .product(name: "Leaf", package: "leaf"),{{/leaf}}
                 .product(name: "Vapor", package: "vapor")
-            ],
-            swiftSettings: [
```

Comment:
Why was this removed? 🤔

Reply:
https://github.com/vapor/template/pull/104#discussion_r1267195853 ROFL great minds.